modules/gapi/misc/python/package/gapi/__init__.py

```diff
@@ -287,15 +287,4 @@ def kernel_with_params(cls):
     return kernel_with_params
 
 
-# FIXME: On the c++ side every class is placed in cv2 module.
-cv.gapi.wip.draw.Rect = cv.gapi_wip_draw_Rect
-cv.gapi.wip.draw.Text = cv.gapi_wip_draw_Text
-cv.gapi.wip.draw.Circle = cv.gapi_wip_draw_Circle
-cv.gapi.wip.draw.Line = cv.gapi_wip_draw_Line
-cv.gapi.wip.draw.Mosaic = cv.gapi_wip_draw_Mosaic
-cv.gapi.wip.draw.Image = cv.gapi_wip_draw_Image
-cv.gapi.wip.draw.Poly = cv.gapi_wip_draw_Poly
-
-cv.gapi.streaming.queue_capacity = cv.gapi_streaming_queue_capacity
-
 cv.gapi.wip.GStreamerPipeline = cv.gapi_wip_gst_GStreamerPipeline
```

Comment:
Still required because `GStreamerPipeline` is registered as `cv.gapi.wip.gst.GStreamerPipeline`

Reply:
/cc @TolyaTalamanov FYI

Reply:
@OrestChura do you know why `GStreamerPipeline` can be just in `cv.gapi.wip.gst` (not `cv.gapi.wip`) ?
Is it because we have alias in c++?

Reply:
@TolyaTalamanov Yes, we have `cv::gapi::wip::GStreamerPipeline` alias, but the initial declaration is in `cv::gapi::wip::gst`.
But it is not necessary to have it in Python, I think.

We can discuss and if needed fix it in a separate PR